src/check-project/utils.js

```diff
@@ -247,8 +247,8 @@ export function constructManifest (manifest, manifestFields, repoUrl, homePage =
     },
     keywords: manifest.keywords ? manifest.keywords.sort() : undefined,
     engines: {
-      node: '>=16.0.0',
-      npm: '>=7.0.0'
+      node: '>=18',
+      npm: '>=9'
```

Comment:
According to the [node releases list](https://nodejs.org/en/download/releases/), only 18.14.0 comes with npm 9, all previous versions come with npm 8 so if it's necessary to mandate npm 9 (is it?) let's make this explicit so people don't have to go looking up why they are getting errors and working out which version is compatible with what.

```suggestion
      node: '>=18.14.0',
      npm: '>=9.3.1'
```